src/server.c

```diff
@@ -2612,6 +2616,7 @@ void createSharedObjects(void) {
     shared.getack = createStringObject("GETACK",6);
     shared.special_asterick = createStringObject("*",1);
     shared.special_equals = createStringObject("=",1);
+    shared.redacted = makeObjectShared(createStringObject("(redacted)",10));
```

Comment:
maybe `******` is more appropriate? just like the password box in web page, not sure.

Reply:
i don't care too much, but i think "(redacted)" is slightly better (can be used to grep, and won't hit some lines with ***), i think it makes it clearer that it was done on purpose by redis and not something else.

Reply:
I agree with Oran, without context ***** is more ambiguous.